### PR TITLE
Reviewed interfaces

### PR DESCRIPTION
**Pull request by wayneeseguin**
This chapter was intense! <3 it!!! 

I have a huge NERDON after reviewing it also! ;)

### CODE REVIEW

File: interfaces.rst [hunk 1]
```diff
@@ -1,25 +1,27 @@
 .. include:: common.txt
 
+.. Interfaces: the awesomesauce of Go
```
Comment (initpy) [inline interfaces.rst:3]: Oh! Absolutely! I'm so changing this title!

File: interfaces.rst [hunk 2]
```diff
@@ -27,82 +29,88 @@ both *know* how to say "Hi".
...
-    type Employee struct{
+    type Employee struct {
         Human //an anonymous field of type Human
         company string
         money float32
     }
 
-    //A human method to say hi
-    func (h *Human) SayHi(){
+    // A human likes to stay... err... *say* hi
```
Comment (initpy) [inline interfaces.rst:69]: Good one! :p

File: interfaces.rst [hunk 3]
```diff
@@ -232,25 +251,27 @@ Output:
...
+As you may have noticed, interfaces types are abstract in that they don't
+themselves implement a given and precise data structure or method.
+They simply say: "if something can do *this*, it may be used *here*".
 
-Also, notice that types implementing a given interface do this simply by
-implementing the methods specified in the interface, they don't *claim*
-explicitely that they are implementing this interface.
+Also, take note that types implementing a given interface do so by simply
+implementing the methods specified by the interface, they don't claim
+that they implement the methods.
```
Comment (initpy) [inline interfaces.rst:260]: They don't claim that they implement (or satisfy) the _interface_ actually. I mean they do implement some methods, but they make no reference at all to a given interface. See what I mean? Hum... I need more English powa to explain it better, I think :)

**Comment by initpy** [comment]
So happy that you liked this chapter! And you made it even better! Thank you!

File: interfaces.rst (hunk 3 above)
Comment (initpy) [inline interfaces.rst:260]: What I mean is: these types make no mention of any interface. Their implementation doesn't explicitly mention a given interface.

Comment (wayneeseguin) [inline interfaces.rst:260]: Ok so this particular paragraph needs rewording still.

**Pull request merged**